terraform/variables.tf

```diff
@@ -43,12 +38,27 @@ variable "subnets" {
   description = "Subnets to use for deployment"
 }
 
-variable "tags" {
```

Comment:
How do we set accounting, owner, etc tags? I don't see `default_tags` in the AWS provider config either.

Reply:
As per the comments in the PR:

> The Terraform in this repository has been heavily modified. It is no longer recommended for you to cd into the Key-Conjurer directory and deploy the terraform by invoking make deploy. We instead suggest that you use our Terraform code as a Terraform module, like so:

You would do this by doing this:

```hcl
provider "aws" {
	default_tags = ...
}

module "keyconjurer-dev" {
	source = "./Key-Conjurer/terraform"
	providers {
		aws = aws.default
	}
}```

Which is what our internal repo which handles the deployment of this project does.